app/Resources/translations/messages.it.yml

```diff
@@ -47,7 +47,6 @@ footer:
     mozilla-ch: Mozilla Switzerland:
 ndg:
     url: https://www.nachrichtendienstgesetz.ch/
-    url2: https://ndg-referendum.ch/
```

Comment:
Removing url2 breaks more than it fixes, since the site doesn't hide the second link if its url is not set. Not even sure we could do that.